tests/functional/tests/rgw/test_rgws.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+
+class TestRGWs(object):
+
+    @pytest.mark.no_docker
+    def test_ceph_rgw_package_is_installed(self, node, Package):
+        # ceph-radosgw is the package name on centos, radosgw is the name on ubuntu
+        has_ceph_radosgw = Package("ceph-radosgw").is_installed
+        has_radosgw = Package("radosgw").is_installed
+        assert (has_ceph_radosgw or has_radosgw)
+
+    def test_rgw_services_are_running(self, node, Service):
+        service_name = "ceph-radosgw@rgw.ceph-{hostname}".format(
```

Comment:
yeah, ceph-ansible provides it's own systemd for the containers. It looks like it's ``ceph-rgw@$hostname``

https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-rgw/tasks/docker/start_docker_rgw.yml#L12